File: 7-kyu/credit-card-checker/javascript/solution.js
const validCard = card => {
  const result = card.replace(/ /g, "")
    .split("")
    .reverse()
    .map(Number)
    .map((digit, idx) => idx % 2 === 1 ? digit * 2 > 9 ? digit * 2 - 9 : digit * 2 : digit)
    .reduce((acc, nxt) => acc + nxt, 0);
  return result % 10 === 0;
};

File: 7-kyu/credit-card-checker/python/solution.py
def valid_card(card):
    digits = [int(d) for d in card.replace(' ', '')]
    doubled = [(digit * 2 - 9 if digit * 2 > 9 else digit * 2) if idx % 2 == 1 else digit for idx, digit in enumerate(reversed(digits))]
    return sum(doubled) % 10 == 0
